install/charts/templates/crd-karydia-security-policy.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2019 SAP SE or an SAP affiliate company. All rights reserved.
+# This file is licensed under the Apache Software License, v. 2 except as
+# noted otherwise in the LICENSE file.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: karydiasecuritypolicies.{{ .Values.metadata.apiGroup }}
+spec:
+  group: {{ .Values.metadata.apiGroup }}
+  version: v1alpha1
+  scope: Cluster
+  names:
+    kind: KarydiaSecurityPolicy
```

Comment:
Let's check if this has to be updated to the new karydiaConfig resource. @SAPJonasZengerle changed the CRD.

Reply:
Changed to current version.